docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
+
+Choose appropriate resources depending on what kind of help you need:
+
+- Documentation sites
+  - Various information
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
+
+The following sections are detailed descriptions of each resource.
+
+## Documentation sites
+
+[Docs guide](docs-guide.md) shows the list of useful documentation sites.
+Visit them and see if there is any information related to your problem.
+
+Note that the documentation sites aren't always latest and perfect.
+If you find some information is wrong, unclear, or missing in Autoware docs, feel free to submit a pull request following our [contribution guidelines](../contributing/index.md).
+
+!!!warning
+
+    Since this documentation site is still under construction, there are some empty pages.
+
+## GitHub Discussions
+
+If you have a problem with Autoware, check existing issues and questions and search similar issues first.
+
+- [Issues](https://github.com/autowarefoundation/autoware/issues)
+
+  Note that Autoware has multiple repositories listed in [autoware.repos](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos).  
+  It is recommended to search across the repositories.
+
+- [Questions](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+
+If no answer was found, create a new question thread [here](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+If your question is not answered in a timely manner (about a week), mention maintainers and remind them.
+
+Also, there are other discussion types such as [feature requests](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests) or [design discussions](https://github.com/autowarefoundation/autoware/discussions/categories/design).
+Feel free to open or join such discussions.
+
+## GitHub Issues
+
+If you have a problem and you have confirmed it is a bug, find the appropriate repository and create a new issue there.
+If you can't determine the appropriate repository, you should not create an issue yet.
+
+!!!warning
+
+    Do not create issues for questions or unconfirmed bugs. If such issues are created, maintainers will transfer them to GitHub Discussions.
+
+If you want to fix the bug by yourself, discuss the approach with maintainers and submit a pull request.
+
+## Slack
+
+Autoware has a Slack workspace for casual communication between contributors.
```

Comment:
Yes, but now it is not `autoware-auto` anymore. So I think we need another channel.

Reply:
By the way, I feel like it excludes newbies from the word `committers`.
So I personally prefer `contributors-autoware`, or just `autoware` looks good to me.

Reply:
Also, I believe discussions between "committers" should be done on GitHub because not all developers join Slack.
They can communicate on Slack, but they should write the conclusion on GitHub as well.
I mean, decision makings shouldn't be closed inside Slack.

It's so important for the community.

Reply:
You raise some good points. The committers Slack channel is currently used by members of the ASWG. At the next TSC there will be a discussion about the Slack vs. Discord proposal from Fatih. We can wait to hear the outcome of that proposal before making changes to the Slack channels.

Reply:
I see, thank you!

> At the next TSC there will be a discussion about the Slack vs. Discord proposal from Fatih.

@BonoloAWF cc @xmfcx @mitsudome-r I have one thing to ask you to consider there.
TIER IV is thinking of connecting the Slack workspace between our company and the Autoware Developer's Community to make more collaborations.
In that case, I think it would be a great benefit for the community because we believe we can provide quick responses to the posts.

So, I'd appreciate it if you could take that into consideration and decide our communication platform. :pray:

Reply:
@kenji-miyake I think it's already been decided, we were just waiting for the right moment to do it (do it with transition to github).
And the right way to announce it to the community.
I've started the ground-work for the migration.

Maybe you can set up a discord or slack bot to connect it to your platform but I suggest you switch to the discord as a company too, we've done it in the past and everyone is happy about it.

Here is the slide I've shared in the TSC call: https://docs.google.com/presentation/d/1hHyD-cWCOoO7hUfhm4ws5eL68HoRT2MzNYZSy-VVsL4/edit?usp=sharing

Reply:
@xmfcx Oh, really? If it's the decision by TSC, it's okay for me.
However, please note that we should consider "fairness of information" as said above.
For example, we should take care not to close discussions only within voice channels.

> I suggest you switch to the discord as a company too

Unfortunately, it has a hurdle. But I'll try it as much as possible for the community.

Reply:
> However, please note that we should consider "fairness of information" as said above.
> For example, we should take care not to close discussions only within voice channels.

Don't worry about that, any decision made will end up in an issue before it is committed as an end result.

> @xmfcx Oh, really? If it's the decision by TSC, it's okay for me.

Mainly no one opposed and many participants supported the move, including with the multiple account requirement for each slack server too as a reason.

Discord will lower the barrier of entry for the new comers for this reason too. They don't need to re-register just for autoware.

Reply:
@xmfcx Will you update this to `Discord` now? Or at a later date?

Reply:
@kenji-miyake we can keep it like this for now, I will create a PR for it later on.

Reply:
Okay, thank you!

Reply:
https://github.com/autowarefoundation/autoware-documentation/issues/7#issuecomment-1062895769